docs/nidigital/class.rst

```diff
@@ -4013,7 +4013,10 @@ serial_number
 
     .. py:attribute:: serial_number
 
-        
+        .. tip:: This property can use repeated capabilities. If set or get directly on the
+            nidigital.Session object, then the set/get will use all repeated capabilities in the session.
+            You can specify a subset of repeated capabilities using the Python index notation on an
+            nidigital.Session repeated capabilities container, and calling set/get value on the result.
```

Comment:
This is so confusing. We need to at least say _which_ repeated capabilities container this can be used on. The vast majority of clients won't call this with `instruments["PXI1Slot3"]` anyway.

Shreyas do we have an issue for improving this documentation?

Reply:
https://github.com/ni/nimi-python/issues/1338